index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css">
+    <link href="https://fonts.googleapis.com/css?family=Tinos&display=swap" rel="stylesheet">
+    <title>Apple Demo</title>
+  </head>
+  <body>
+    
+    <nav>
+      <a href="#" class="fab fa-apple"></a>
+      <a href="#">Store</a>
+      <a href="#">Mac</a>
+      <a href="#">ipod</a>
+      <a href="#">iphone</a>
+      <a href="#">ipad</a>
+      <a href="#">itunes</a>
+      <a href="#">Support</a>
+      <input type="text" placeholder="Search here">
+
+    </nav>
+    <div class="story">
+      <h1>What will your verse be?</h1>
+      <p>Everyone has something to share. See how people are using  iPad in their <br>unique ways to add to the world’s story.</p>
+      <span>Watch the “Your Verse” film and explore the stories <i class="fas fa-play-circle"></i></span>
+       
+  </div>
```

Comment:
Let us improve semantics by using ```<main>``` instead of ```<div class="story">```

Reply:
I changed it